src/constant/network.ts

```diff
@@ -24,8 +24,11 @@ export const CarbonEvmChainIDs = {
 
 export const DEFAULT_NETWORK = Network.MainNet;
 
-export interface EthNetworkConfig {
-  rpcURL: string;
+export interface BaseNetworkConfig {
```

Comment:
Maybe rename this to `BasicNetworkConfig` so there's no confusion with Base network